#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("tolerance", double_t, 0, "margin length of each clustering", 0.02, 0.0, 1.0)
gen.add("max_size", int_t, 0, "the max number of the points of each cluster", 25000, 0, 2000000)
gen.add("min_size", int_t, 0, "the minimum number of the points of each cluster", 20, 0, 1000)
gen.add("downsample", bool_t, 0, "downsample pointcloud for clustering or not", False)
gen.add("leaf_size", double_t, 0, "downsampling leaf_size", 0.01, 0.0, 1.0)


exit (gen.generate (PACKAGE, "jsk_pcl_ros", "MultiEuclideanClustering"))
